packages/ses/src/main.js

```diff
@@ -12,4 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import Compartment from './compartment-shim.js';
+
+export { Compartment };
```

Comment:
Can these be merged as with lockdown-shim?

```js
export { Compartment } from './lockdown-shim.js';
```

Reply:
Is that the syntax to use when `Compartment` is the default export of `lockdown-shim`? I wasn't sure.

Reply:
Pardon, fixed the typo in the example, but also disregard. I didn’t realize that there was a difference in the way lockdown-shim exports by name and compartment-shim exports default. I’m new to the ESM system myself. I’m wondering whether `export Compartment from './compartment-shim.js'` would work for the default case.

Reply:
Nope. This works though:

```js
export { default as Compartment } from './compartment-shim.js';
```